distribution/docs/src/main/resources/content/_tables/Session.adoc

```diff
@@ -19,7 +19,7 @@
 |Session Timeout (in minutes)
 |expirationTime
 |Integer
-|The number of minutes after a session has been inactive that it should be invalidated.
+|Specifies the length of inactivity (in minutes) that will cause a user to be logged out automatically. This value must be 2 minutes or greater, as users are warned when only 1 minute remains. If a value of less than 2 minutes is used, the timeout is set to the default time of 31 minutes.
```

Comment:
❓ What is session timeout? Does this happen on the backend? Could this description be improved to explain the practical consequences of this timeout? Does it cause SSH consoles to be automatically exited? Does it cause users to be logged out in Intrigue/Admin UI? Why would someone want to change this? I see this note in the documentation:
https://github.com/codice/ddf/blob/080dea730aa7c06240cb97e74cc5b43203242242/distribution/docs/src/main/resources/content/_configuring/environment-hardening.adoc#L17-L18
https://github.com/codice/ddf/blob/080dea730aa7c06240cb97e74cc5b43203242242/distribution/docs/src/main/resources/content/_configuring/environment-hardening.adoc#L96-L104
Is this configuration related to https://github.com/codice/ddf/blob/080dea730aa7c06240cb97e74cc5b43203242242/distribution/docs/src/main/resources/content/_tables/platform.ui.config.adoc#L75-L80?

Reply:
I'm not sure whether those hardening steps are actually related to the config you referenced @emmberk. It looks like based on the steps the hardening guide is only referencing the `org.codice.ddf.security.filter.login.Session` config but not the `platform.ui.config`

Reply:
The ticket was written to help prevent confusion about all of these different configurations, right? I guess what I'm saying is that I'm still confused. I don't think these current updates are sufficient. But I might be missing something 🤷‍♀️

Reply:
Looking at the JavaDoc of the HttpSession, which this expirationTime value is used for:
`Specifies the time, in seconds, between client requests before the servlet container will invalidate this session.`
So this should apply to all client sessions, not limited to just Intrigue. That's something we should make clear.

Also, the description of `If a value of less than 2 minutes is used, the timeout is set to the default time of 31 minutes.` is not enforced by the code. See https://github.com/codice/ddf/blob/2.13.x/platform/security/filter/security-filter-login/src/main/java/org/codice/ddf/security/filter/login/LoginFilter.java#L707-L713

Reply:
Appended PR 
@emmberk I believe it is less confusing now.
@peterhuffer It is now enforced by the code.